data/adapters/test_continuous_adapter.py

```diff
@@ -169,6 +169,7 @@ def test_interpret_instructions_perturb(self, mock_perturb):
         mock_self = mock.Mock(spec=continuous_adapter.StandardizingAdapter)
         mock_self.perturb_ratio = 0.5
         mock_self.rescale_ratio = None
+        mock_self.random_generator = None
```

Comment:
What's the rationale for not using a random_generator here? I don't recall how this functions currently in the ContinuousAdapter class.

Reply:
Nevermind, I think I got why you're doing this in these tests.